src/constants/WalletAndCurrencyConstants.js

```diff
@@ -53,6 +53,7 @@ export const CURRENCY_PLUGIN_NAMES = {
   SMART: 'smartcash',
   UFO: 'ufo',
   VTC: 'vertcoin',
+  XLM: 'stellar',
   XZC: 'zcoin'
```

Comment:
Since the currencyName (based on `currencyCode`) is a property whose subproperties are accessed in the `mergeSettings` function, that means missing currencies in this object will cause an error (which is why I'm filling in Stellar).